authority/admin/db/nosql/policy.go

```diff
@@ -0,0 +1,121 @@
+package nosql
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"go.step.sm/linkedca"
+
+	"github.com/smallstep/certificates/authority/admin"
+	"github.com/smallstep/nosql"
+)
+
+type dbAuthorityPolicy struct {
+	ID          string           `json:"id"`
+	AuthorityID string           `json:"authorityID"`
+	Policy      *linkedca.Policy `json:"policy"`
+}
+
+func (dbap *dbAuthorityPolicy) convert() *linkedca.Policy {
+	if dbap == nil {
+		return nil
+	}
+	return dbap.Policy
+}
+
+func (db *DB) getDBAuthorityPolicyBytes(ctx context.Context, authorityID string) ([]byte, error) {
+	data, err := db.db.Get(authorityPoliciesTable, []byte(authorityID))
+	if nosql.IsErrNotFound(err) {
+		return nil, admin.NewError(admin.ErrorNotFoundType, "authority policy not found")
+	} else if err != nil {
+		return nil, fmt.Errorf("error loading authority policy: %w", err)
```

Comment:
True, it's a bit weird. It's the same for the other admin DB operations, though. 

Right now errors will be wrapped when handling them. The `ErrorNotFound` case will be ignored and we'll check for nil, instead. This could be simplified to returning just nil in case `nosql.IsErrNotFound` is true. That would fit the other case where I propose to do so, but we would have to agree on going forward like that.